functors/README.md

```diff
@@ -0,0 +1,13 @@
+# Functors
+
+This directory contains examples of _functors_ with various strategies for distributed evaluation.
+
+[Locally evaluated identity functor](identity-functor.amb), roughly equivalent to following JS code:
+
+```javascript
+let identity = (value) => () => ({ value })
+let map_identity = () => (id, func) => identity(func(id().value))
```

Comment:
Can we rename "func" to something else to not confuse the argument name with func primitive in the encoding? Eg. "f" of "fn"?